### DISCUSSION TITLE
How to find Trust Wallt wallet word combination password?

### DISCUSSION THREAD

**Discussion by mehmet-krts** [community/community#121634] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello Everyone, I was investing in Trust Wallt wallet crypto, I did not save the wallet word combination password, thinking that there would be no problem. There was a problem with the phone, it did not turn on, I had to format it and I could not log in to the wallet again. Can a word combination password be found with the wallet account number? I ask for your support.

**Comment by LiteBrite82** [comment]
Hi @mehmet-krts ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses Trust Wallet, such as the [Trust Wallet support page](https://support.trustwallet.com/en/support/home). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

I'm going to move this Discussion over to the General category for greater visibility 

Best of luck!

**Reply by mehmet-krts** [reply]
İlginiz için teşekkür ederim.
